src/gcp/cloudfunctionsv2.ts

```diff
@@ -104,6 +111,7 @@ export interface ServiceConfig {
   timeoutSeconds?: number;
   availableMemory?: string;
   environmentVariables?: Record<string, string>;
+  secretEnvironmentVariables?: Array<SecretEnvVar>;
```

Comment:
Nit: `SecretEnvVar[]` ?

Reply:
huh I finally read go/tsstyle#arrayt-type. Sounds like that is perferable.